src/core/tools/tool_helper_restrict.cpp

```diff
@@ -41,6 +53,9 @@ std::string ToolHelperRestrict::restrict_mode_to_string() const
     case RestrictMode::Y:
         return "Y only";
         break;
+    case RestrictMode::DEG45:
+        return "45 degrees only";
```

Comment:
Space in the tool bar is quite limited, I'd use the ° sign:
```suggestion
        return "45° only";
```